vdr/vdr.go

```diff
@@ -127,7 +133,22 @@ func (r VDR) Resolve(id did.DID, metadata *types.ResolveMetadata) (*did.Document
 
 // Update updates a DID Document based on the DID and current hash
 func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, metadata *types.DocumentMetadata) error {
```

Comment:
why is `metadata` a pointer? Also, why isn't it used? (drop it?)

Reply:
It's part of the interface which is the same as the storage backend. Not sure yet what to do with it. But this implementation is not finished yet.